### DISCUSSION TITLE
انتقال بکاپ از پنل مرزبان؟

### DISCUSSION THREAD

**Discussion by DrAmirB** [GFWFuckers/MarzGosha#18] [Q&A]
سلام وقت بخیر آیا امکان انتقال بکاپ مرزبان به این پنل  وجود داره و مشکلی پیش نمیاد؟

**Comment by xGavazn** [comment]
> سلام وقت بخیر آیا امکان انتقال بکاپ مرزبان به این پنل وجود داره و مشکلی پیش نمیاد؟

میتونید طبق این [آموزش](https://github.com/GFWFuckers/MarzGosha/discussions/27) جلو برید https://github.com/GFWFuckers/MarzGosha/discussions/27
